format.go

```diff
@@ -42,14 +43,16 @@ func writeFolded(w io.Writer, s map[string]int) error {
 	return nil
 }
 
-func toPprof(s map[string]int, hz int) *profile.Profile {
+func toPprof(s map[string]int, hz int, startTime, endTime time.Time) *profile.Profile {
 	functionID := uint64(1)
 	locationID := uint64(1)
 	line := int64(1)
 
 	p := &profile.Profile{}
 	m := &profile.Mapping{ID: 1, HasFunctions: true}
 	p.Period = int64(1e9 / hz) // Number of nanoseconds between samples.
+	p.TimeNanos = startTime.UnixNano()
+	p.DurationNanos = endTime.UnixNano() - p.TimeNanos
```

Comment:
🐞 This is going to produce incorrect results when clock changes or leap seconds are involved. I'll push a fix.